test/pspm_convert_ecg2hb_test.m

```diff
@@ -117,33 +117,36 @@ function invalid_input(this, filename, chan_struct)
       this.verifyWarning(@()pspm_convert_ecg2hb(filename, 'bla'), 'ID:invalid_input');
       if chan_struct.nr ~= 1
         % invalid channel
-        this.verifyWarning(@()pspm_convert_ecg2hb(filename, 1), 'ID:unexpected_channeltype');
+        this.verifyWarning(@()pspm_convert_ecg2hb(filename, struct('channel', 1)), 'ID:unexpected_channeltype');
       end
       % invalid twthresh (text)
-      o.twthresh = 'bla';
-      this.verifyWarning(@()pspm_convert_ecg2hb(filename, chan_struct, o), 'ID:invalid_input');
+      o = struct('twthresh', 'bla');
```

Comment:
Similarly to the other test function, could be useful to change this to `o = struct('channel', 1, 'twthresh', 'bla')`. However it looks like the following tests are looking at the correctness of other variables, this suggestion may not be useful.